services/app-web/src/pages/StateSubmission/ContractDetails/ContractDetails.tsx

```diff
@@ -271,232 +217,359 @@ export const ContractDetails = ({
+                    ? undefined
+                    : draftSubmission.draftRevision.formData
+                          .modifiedPaymentsForMentalDiseaseInstitutions
+            ) ?? '',
+        modifiedMedicalLossRatioStandards:
+            booleanAsYesNoFormValue(
+                draftSubmission.draftRevision.formData
+                    .modifiedMedicalLossRatioStandards === null
+                    ? undefined
+                    : draftSubmission.draftRevision.formData
+                          .modifiedMedicalLossRatioStandards
+            ) ?? '',
+        modifiedOtherFinancialPaymentIncentive:
+            booleanAsYesNoFormValue(
+                draftSubmission.draftRevision.formData
+                    .modifiedOtherFinancialPaymentIncentive === null
+                    ? undefined
+                    : draftSubmission.draftRevision.formData
+                          .modifiedOtherFinancialPaymentIncentive
+            ) ?? '',
+        modifiedEnrollmentProcess:
+            booleanAsYesNoFormValue(
+                draftSubmission.draftRevision.formData
+                    .modifiedEnrollmentProcess === null
+                    ? undefined
+                    : draftSubmission.draftRevision.formData
+                          .modifiedEnrollmentProcess
+            ) ?? '',
+        modifiedGrevienceAndAppeal:
+            booleanAsYesNoFormValue(
+                draftSubmission.draftRevision.formData
+                    .modifiedGrevienceAndAppeal === null
+                    ? undefined
+                    : draftSubmission.draftRevision.formData
+                          .modifiedGrevienceAndAppeal
+            ) ?? '',
+        modifiedNetworkAdequacyStandards:
+            booleanAsYesNoFormValue(
+                draftSubmission.draftRevision.formData
+                    .modifiedNetworkAdequacyStandards === null
+                    ? undefined
+                    : draftSubmission.draftRevision.formData
+                          .modifiedNetworkAdequacyStandards
+            ) ?? '',
+        modifiedLengthOfContract:
+            booleanAsYesNoFormValue(
+                draftSubmission.draftRevision.formData
+                    .modifiedLengthOfContract === null
+                    ? undefined
+                    : draftSubmission.draftRevision.formData
+                          .modifiedLengthOfContract
+            ) ?? '',
+        modifiedNonRiskPaymentArrangements:
+            booleanAsYesNoFormValue(
+                draftSubmission.draftRevision.formData
+                    .modifiedNonRiskPaymentArrangements === null
+                    ? undefined
+                    : draftSubmission.draftRevision.formData
+                          .modifiedNonRiskPaymentArrangements
+            ) ?? '',
+        statutoryRegulatoryAttestation:
+            booleanAsYesNoFormValue(
+                draftSubmission.draftRevision.formData
```

Comment:
how come we don't just make booleanAsYesNoFormValue work with null instead of doing this check everywhere?